articles/azure-vmware/enable-managed-snat-for-workloads.md

```diff
@@ -17,12 +17,8 @@ With this capability, you:
 - Are unable to view connection logs. 
 - Have a limit of 128 000 concurrent connections.  
 
-## Prerequisites
-- Azure Solution VMware private cloud
-- DNS Server configured on the NSX-T Datacenter
-
 ## Reference architecture
-The architecture shows Internet access to and from your Azure VMware Solution private cloud using a Public IP directly to the NSX Edge.     
+The architecture shows Internet access outbound from your Azure VMware Solution private cloud using an AVS Managed SNAT Service.     
```

Comment:
@yash177 please spell out 'AVS' as 'Azure VMware Solution'. We can't use an acronym for it as one is not legally approved. Thanks